### PR TITLE
Fix/milestone status update

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
ZH: https://github.com/bcgov/cas-cif/issues/825

### CODE REVIEW

File: app/components/Form/ProjectMilestoneReportForm.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
   paymentSchema,
   paymentUiSchema,
 } from "data/jsonSchemaForm/paymentSchema";
-import { ProjectMilestoneReportForm_reportingReqiurement$key } from "__generated__/ProjectMilestoneReportForm_reportingReqiurement.graphql";
+import { ProjectMilestoneReportForm_reportingRequirement$key } from "__generated__/ProjectMilestoneReportForm_reportingRequirement.graphql";
```
Comment (BCerki) [inline app/components/Form/ProjectMilestoneReportForm.tsx:19]: Do you know why this is showing as diffs? This file looks exactly the same to me

**Review by BCerki — COMMENTED** [review]
This PR may have introduced a new bug: https://github.com/bcgov/cas-cif/issues/908 (although the changes don't seem obviously connected). I'm not able to reproduce the bug on `dev`, only on this branch.

**Comment by Sepehr-Sobhani** [comment]
> 

We have discussed this issue yesterday with @pbastia and @dleard on card https://github.com/bcgov/cas-cif/issues/836
This seems to be the issue with dev data so when we create a new milestone report and then delete it after project submission, it gets deleted as it should.

**Comment by BCerki** [comment]
> > 
> 
> We have discussed this issue yesterday with @pbastia and @dleard on card #836 This seems to be the issue with dev data so when we create a new milestone report and then delete it after project submission, it gets deleted as it should.

I found the bug when I made my own milestone report (in the loom you can see the description is bri), but I just tried to reproduce again and couldn't

**Comment by Sepehr-Sobhani** [comment]
> > > 
> > 
> > 
> > We have discussed this issue yesterday with @pbastia and @dleard on card #836 This seems to be the issue with dev data so when we create a new milestone report and then delete it after project submission, it gets deleted as it should.
> 
> I found the bug when I made my own milestone report (in the loom you can see the description is bri), but I just tried to reproduce again and couldn't

hmmm, weird 🤔

**Review by BCerki — APPROVED** [review]

**Review by pbastia — CHANGES_REQUESTED** [review]
Very nice! We're just missing a test case in `schema/test/unit/computed_columns/project_revision_milestone_report_statuses_test.sql`

**Review by pbastia — APPROVED** [review]

**Pull request merged**